packages/selinux-policy/rules.cil

```diff
@@ -26,6 +26,11 @@
 ; Unprivileged components cannot interact with privileged processes.
 (neverallow unprivileged_s privileged_s (processes (interact)))
 
+; Never allow noatsecure to any subject
+(neverallow all_s all_s (process (noatsecure)))
+; Don't audit avc denials for a process that attepts to use noatsecure
```

Comment:
typo:
```suggestion
; Don't audit avc denials for a process that attempts to use noatsecure
```

However, the comment is misleading. It's less about the process actually attempting to use `noatsecure`, and more about the SELinux LSM choosing to apply `AT_SECURE` on domain transitions unless the policy allows `noatsecure`:
https://elixir.bootlin.com/linux/v6.2/source/security/selinux/hooks.c#L2380